common/src/rv_trace.rs

```diff
@@ -355,6 +358,14 @@ impl ELFInstruction {
             RV32IM::VIRTUAL_ASSERT_VALID_DIV0,
         );
 
+        // All instructions in virtual sequence are mapped from the same
+        // ELF address. Thus if an instruction is virtual (and not the last one
+        // in its sequence), then we should *not* update the PC.
+        flags[11] = match self.virtual_sequence_remaining {
+            Some(i) => i != 0,
+            None => false
+        };
```

Comment:
We need this flag (and so we had to reverse the virtual sequence indexing) for the new non-uniform constraint, which ensures that virtual sequences are executed in full and in order